docs/content/guides/developer/sui-101/create-coin.mdx

```diff
@@ -4,26 +4,177 @@ title: Create a Coin
 
 Publishing a coin on Sui is nearly as straightforward as publishing a new type. The main difference is the requirement of a [one-time witness](/concepts/sui-move-concepts/one-time-witness.mdx) when creating a coin.
 
-```move
+```rust
 module examples::mycoin {
     use std::option;
-    use sui::coin;
+    use sui::coin::{Self, Coin, TreasuryCap};
     use sui::transfer;
     use sui::tx_context::{Self, TxContext};
 
     /// The type identifier of coin. The coin will have a type
     /// tag of kind: `Coin<package_object::mycoin::MYCOIN>`
     /// Make sure that the name of the type matches the module's name.
-    struct MYCOIN has drop {}
+    struct MYCOIN has drop {};
 
     /// Module initializer is called once on module publish. A treasury
     /// cap is sent to the publisher, who then controls minting and burning
     fun init(witness: MYCOIN, ctx: &mut TxContext) {
         let (treasury, metadata) = coin::create_currency(witness, 6, b"MYCOIN", b"", b"", option::none(), ctx);
         transfer::public_freeze_object(metadata);
-        transfer::public_transfer(treasury, tx_context::sender(ctx))
+        transfer::public_transfer(treasury, tx_context::sender(ctx));
     }
 }
 ```
 
 The `Coin<T>` is a generic implementation of a coin on Sui. The owner of the `TreasuryCap` gets control over the minting and burning of coins. Further transactions can be sent directly to the `sui::coin::Coin` with `TreasuryCap` object as authorization.
+
+Extending the example further, add a `mint` function to the module. Use the `mint_and_transfer` function of the `Coin` module to create (mint) a coin and transfer it to an address. 
+
+```rust
+public fun mint(
+        treasury_cap: &mut TreasuryCap<MYCOIN>, amount: u64, recipient: address, ctx: &mut TxContext
+    ) {
+        coin::mint_and_transfer(treasury_cap, amount, recipient, ctx)
+    }
+``` 
+
+### Sui CLI
+
+If you published the previous example to a Sui network, you can use the `sui client call` command to mint coins and deliver them to the address you provide. See [Sui CLI](../../../references/cli.mdx) for more information on the command line interface.
+
+```shell
+sui client call --function mint --module coinage --package <PACKAGE-ID> --args <TREASURY-CAP-ID> <NUMBER-OF-COINS> <RECIPIENT-ADDRESS> --gas-budget <GAS-AMOUNT>
+```
+
+## DenyList
+
+:::info
+
+The `DenyList` object and regulated coins are currently available in Devnet only. 
+
+:::
+
+The Sui framework provides a `DenyList` singleton, shared object that the bearer of a `DenyCap` can use to specify a list of addresses that are unable to use the specified object. This is useful, for example, when creating a regulated coin on Sui that requires the ability to block certain addresses from using it as inputs to transactions. Regulated coins on Sui satisfy any regulations that require the ability to prevent known bad actors from having access to those coins. 
```

Comment:
```suggestion
The Sui framework provides a `DenyList` singleton, shared object that the bearer of a `DenyCap` can use to specify a list of addresses that are unable to use coins of the specified type. This is useful, for example, when creating a regulated coin on Sui that requires the ability to block certain addresses from using it as inputs to transactions. Regulated coins on Sui satisfy any regulations that require the ability to prevent known bad actors from having access to those coins. 
```
> that are unable to use coins of the specified type

I'm struggling with this part here that I edited because in general the DenyList can be used for any core type defined in `sui`, but right now it's limited to coins